### PR TITLE
! fix runtime on ARM32

### PR DESCRIPTION
**Pull request by eloraiby**
caused by the use of unsigned char instead of signed char : now using stdint.h to define types

### CODE REVIEW

File: src/DNA/native/src/PInvoke.c [hunk 1]
```diff
@@ -95,7 +95,7 @@ fnPInvoke PInvoke_GetFunction(tMetaData *pMetaData, tMD_ImplMap *pImplMap) {
 
 	libName = MetaData_GetModuleRefName(pMetaData, pImplMap->importScope);
 
-#ifndef _WIN32
+#ifndef NO_JS
```
Comment (ncave) [inline src/DNA/native/src/PInvoke.c]: @eloraiby Would you consider renaming this to NO_JS_INTEROP? It's also fine if you prefer to keep it the way it is.
I previously had the same define here named NO_JS_INTEROP but took it out to merge some upstream changes.

**Comment by eloraiby** [comment]
@ncave done! are you planning to push the changes to the other blazor repos ?

**Comment by ncave** [comment]
@eloraiby Yeah, I'll add these to my PR. Thanks!

**Comment by ncave** [comment]
@eloraiby Hopefully you don't mind, converted the js interop to an opt-in flag in #16, it's clearer that way.

**Comment by eloraiby** [comment]
I see no problem with that. I will start working on the amd64 crash now.

**Pull request merged**